torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -547,12 +547,12 @@ Tensor mm_mat2_backward(const Tensor & grad, const Tensor & mat1, IntArrayRef si
       int64_t out_cols = grad.size(1);
       Tensor t = at::zeros({}, grad.options()).expand({out_rows, out_cols}, true);
       Tensor r = at::empty({out_cols, out_rows}, grad.options()).t();
-      at::addmm_out(r, t, mat1.t(), grad, alpha, 1);
+      at::addmm_out(r, t, mat1.conj().t(), grad, alpha, 1);
       return r;
     }
-    return maybe_multiply(grad.t().mm(mat1).t(), alpha);
+    return maybe_multiply(grad.t().mm(mat1.conj()).t(), alpha);
```

Comment:
I wonder if these things would look cleaner if `grad.t().mm(mat1.conj()).t()` were written as `mat1.conj().t().mm(grad)` (and what's more whether `conj().t()` should be `adj()` or `adjoin()` or something).

Reply:
yeah I think that's a good idea. We are planning to add `Hermitian` operator soon so yes again I agree it would be nice to replace `mat1.conj().t()` with `mat1.H()` or `mat1.adj()`. In the meantime, I will do the reordering based on your suggestion.

Reply:
> // if input was column-major, return grad as column-order for efficiency

The reordering would be BC-breaking (the strides of the gradient would change), but that might be OK. We should definitely benchmark the reordering to make sure it's not introducing any regressions.

Reply:
Thanks for spotting that!